cypress/e2e/zenscrape/register.cy.js

```diff
@@ -12,11 +12,11 @@ describe('Zenscrape Register Page  ', () => {
         cy.get('.rounded-md > .shadow').click();
     });
 
-    it('2. Test to redirect to the Register page', () => {
+    it('2. Test to check the to the register page', () => {
         cy.navigateUrlwithCookies('https://app.zenscrape.com/register');
     });
 
-    it('3. Test to Verify subscription text is there or not', () => {
+    it('3. Test to Verify subscription text is present or not', () => {
```

Comment:
or not ?? 
To verify that a "Subscription" text is present

Reply:
removed or not